index.html

```diff
@@ -0,0 +1,122 @@
+           (MON)~(FRI) FROM 08:00am to 5:00pm
+            </p>
+            <p class="bottom-text">@ Meanwood Chamba Valley</p>
+</main>
+    <section class="main-section">
+        <h2 class="main-section-h1">
+            Main Program
+        </h2>
+            <div class="orange-underline"></div>
+                <ul class="blocks">
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Seminars</h2>
+                        <p>we offer seminars to farmers every two months
+                            per year. We attach importance to collabos
+                        </p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Lectures</h2>
+                        <p>Learn mordern farming techniques
+                            in our daily classes from monday to friday
+                            each week
+                        </p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Programs</h2>
+                        <p>Enrol and become eligible for our
+                            government sponsored susidies
+                        </p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Assistance</h2>
+                        <p>Access technical support
+                            form our technical teams
+                            available to you 24 hours a day
+                        </p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Acreditation</h2>
+                        <p>Become acredited with us and gain
+                            the advantages that come with being an
+                            affiliate.
+                        </p>
+                    </li>
+                </ul>
+                <button class="join-btn more-btn" type="button">Attend The Summit</button>
+            <a href="#" class="see-more">See more</a>    
+     </section>
+<section class="speaker-sec">
+    <h2 class="speaker-header">Featured Guests</h2>
+        <div class="orange-underline"></div>
+            <ul class="speaker-list" id="speaker-list">
+               
+            </ul>
+        <button class="more-btn" id="more-btn" type="button">More</button>
+</section>
+<section class="partner">
+    <h2 class="partner-header">Partners</h2>
+        <div class="orange-underline"></div>
+    <ul class="bottom-logos">
+        <li><a href="https://www.usaid.gov/"><img src="img/ebay.png" alt="logo"></a></li>
+        <li><a href="https://www.afro.who.int/countries/zambia"><img src="img/who.svg" alt="logo"></a></li>
+        <li><a href="https://www.unza.zm/"><img src="img/Unza-logo.svg" alt="logo"></a></li>
+        <li><a href="https://www.skype.com/en/"><img src="img/skype.png" alt="logo"></a></li>
+    </ul>
+        
+</section>
```

Comment:
- [ ] The images in this section are not equal in shape. You can check in mobile version how it looks. These are for example. Please check all pictures in this section and fix them.

![image](https://user-images.githubusercontent.com/38635158/124987290-5cd8a800-e045-11eb-8ee1-2959ec96c156.png)